py-polars/polars/io/excel/functions.py

```diff
@@ -196,10 +213,15 @@ def read_excel(
 
     if xlsx2csv_options is None:
         xlsx2csv_options = {}
+
     if read_csv_options is None:
-        read_csv_options = {}
+        read_csv_options = {"truncate_ragged_lines": True}
+    elif "truncate_ragged_lines" not in read_csv_options:
+        read_csv_options["truncate_ragged_lines"] = True
```

Comment:
Won't this incur a performance penalty?

Reply:
Fortunately not - it's close to zero cost here, which is why I don't feel bad enabling it by default ;)